README.md

```diff
@@ -143,8 +143,8 @@ Install and execute binaries from Python packages.
 Binaries can either be installed globally into isolated Virtual Environments
 or run directly in an temporary Virtual Environment.
 
-Virtual Envrionment location is /home/USER/.local/pipx/venvs.
-Symlinks to binaries are placed in /home/USER/.local/bin.
+Virtual Envrionment location is /Users/jaraco/.local/pipx/venvs.
```

Comment:
Can you re-pull from master and regenerate the docs? I updated the readme template to overwrite $HOME with /home/USER.

Reply:
I merged with master in 67c2ac4 and regenerated docs in 8f55741. This branch is up-to-date and doesn't render "/home/USER":

```
pipx feature/tox $ git fetch origin                                                                                                                                            
pipx feature/tox $ git merge origin/master                                                                                                                                     
Already up to date.
pipx feature/tox $ tox -e docs                                                                                                                                                 
docs develop-inst-noop: /Users/jaraco/code/public/pipx
docs installed: action: docs, msg: envreport,cwd: /Users/jaraco/code/public/pipx,cmd: /Users/jaraco/code/public/pipx/.tox/docs/bin/python -m pip freeze,appdirs==1.4.3,attrs==19.1.0,black==19.3b0,Click==7.0,entrypoints==0.3,flake8==3.7.7,Jinja2==2.10.1,MarkupSafe==1.1.1,mccabe==0.6.1,mypy==0.700,mypy-extensions==0.4.1,-e git+gh://pipxproject/pipx@8f55741ff9e5b248afdea9b74bfea62eff8bb732#egg=pipx,pycodestyle==2.5.0,pyflakes==2.1.1,toml==0.10.0,typed-ast==1.3.1
docs run-test-pre: PYTHONHASHSEED='1194062344'
docs run-test: commands[0] | python generate_readme.py
___________________________________________________________________________________ summary ___________________________________________________________________________________
  docs: commands succeeded
  congratulations :)

pipx feature/tox $ git diff                                                                                                                                                    

pipx feature/tox $                
```

Reply:
See #136 for a technique that I think better achieves what you're after.